frontend-react/src/components/Admin/EditReceiverSettings.tsx

```diff
@@ -324,24 +303,41 @@ export function EditReceiverSettings({ match }: RouteComponentProps<Props>) {
                     newjson={orgReceiverSettingsNewJson}
                 />
             </GridContainer>
-        );
-    };
+        </section>
+    );
+};
+
+type Props = {
+    orgname: string;
+    receivername: string;
+    action: "edit" | "clone";
+};
+
+export function EditReceiverSettings({ match }: RouteComponentProps<Props>) {
+    const orgname = match?.params?.orgname || "";
+    const receivername = match?.params?.receivername || "";
+    const action = match?.params?.action || "";
 
     return (
-        <NetworkErrorBoundary
-            fallbackComponent={() => <ErrorPage type="page" />}
+        <AdminFormWrapper
+            header={
+                <>
+                    {`Org name: ${
```

Comment:
This seems like a good candidate to convert to its own component. We have `components/Title` which might serve this page well? It'll create consistency with headers on other pages.

Reply:
Thanks for the tip! I'm all for consistency. Done in  https://github.com/CDCgov/prime-reportstream/pull/5372/commits/ca93c0e050e96f6921da09a9bc50e2f4e40e3fe5. 



<img width="641" alt="image" src="https://user-images.githubusercontent.com/2730338/166727171-cbe4bcf7-a733-4458-b944-e3bb45394ff9.png">

Reply:
Looks great! Thanks